tests/components/geonetnz_quakes/test_config_flow.py

```diff
@@ -1,6 +1,8 @@
 """Define tests for the GeoNet NZ Quakes config flow."""
 from datetime import timedelta
 
+from asynctest import patch
```

Comment:
We should import from `tests.async_mock`

```suggestion
from tests.async_mock import patch
```

(this suggestion needs to be moved down to satisfy isort)

Reply:
Will swap it out for the new hotness 👍

Reply:
Fixed the other occurrences in our codebase #37428